pymc_marketing/clv/models/beta_geo.py

```diff
@@ -92,33 +84,22 @@ class BetaGeoModel(CLVModel):
                 "cores": 2,
             },
         )
-        model.build_model()
-        model.fit()
+        # Fit model quickly to large datasets via the default Maximum a Posteriori method
+        model.fit(fit_method='map')
```

Comment:
There are way more issues in that docstring than just this haha. The docs prebuild preview was very helpful in finding them.

My general writing approach is to be concise, and limit technical jargon as much as possible. Let me know if the updated docstrings pass judgment.

Reply:
Doesn't the ruff formatter change ' to "?

Reply:
@wd60622 I'm not seeing this single-double quote change in the docs preview. Maybe because it's inside of a reStructuredText python block inside of a docstring?